substrate/frame/examples/multi-block-migrations/src/migrations/v1/weights.rs

```diff
@@ -33,7 +33,7 @@
 // --pallet
 // pallet_example_mbm
 // --extrinsic
-// 
+//
```

Comment:
```suggestion
// 
```

So that there are no changes here